Simplenote/src/main/java/com/automattic/simplenote/NotesActivity.java

```diff
@@ -992,6 +1003,15 @@ public void onConfigurationChanged(Configuration newConfig) {
                     mNoteListFragment.getListView().clearChoices();
                 }
                 invalidateOptionsMenu();
+            } else {
+                if (mNoteListFragment.isHidden()) {
```

Comment:
Can this `if` statement can be combined with the `else` above it to make the branch `} else if (mNoteListFragment.isHidden()) {`?

Reply:
Yes I think we can, but it may become a little confusing for other developers. the `else` clause is representing `newConfig.orientation == Configuration.ORIENTATION_PORTRAIT` here. Changing that to `else if (mNoteListFragment.isHidden())` would work, but would mask the fact that the orientation is Portrait in this case from the developer at first glance.

If you still think it is OK to change it, let me know.

Reply:
Also, as for the requested changes concerning the extra spacings, I am unable to comment on that for some weird reason. I think I commented on that yesterday but it didn't go through.

Anyways, I use Android Studio standard formatting in all of my other Android projects and therefore I wouldn't turn that feature off. Would you mind removing those spaces if that's not much trouble?